scripts/zephyr_module.py

```diff
@@ -161,6 +161,24 @@
          type: seq
          sequence:
             - type: str
+  package-managers:
+    required: false
+    type: map
+    mapping:
+      pip:
+        required: false
+        type: map
+        mapping:
+          packages:
+            required: false
+            type: seq
+            sequence:
+              - type: str
+          requirements:
```

Comment:
I think being a bit more verbose here will make this field self-explanatory.
```suggestion
          requirements-files:
```

(not blocking comment)

Reply:
Updated